src/styles/blocks/card.scss

```diff
@@ -0,0 +1,92 @@
+.card {
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+
+  &:nth-child(4) {
+    margin-right: 0;
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+    display: block;
+  }
+
+  &__title {
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    color: $secondary-color;
+    margin: 0 0 4px;
+  }
+
+  &__code {
+    font-weight: 400;
+    font-size: 10px;
+    color: #616070;
+    margin-bottom: 16px;
+    line-height: 14px;
+  }
+
+  &__review {
+    font-weight: 400;
+    font-size: 10px;
+    color: $secondary-color;
+    line-height: 14px;
+  }
+
+  &__name-hint {
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+    color: #616070;
+  }
+
+  &__number {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: $secondary-color;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 0 0 24px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 0 0 16px;
+  }
+
+  &__buy {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 40px;
+    background: $main-color;
+    color: white;
+    text-decoration: none;
+    border-radius: 5px;
+    border: 1px solid $main-color;
+    box-sizing: border-box;
+    font-weight: 700;
+    font-size: 14px;
+    text-transform: uppercase;
+    transition-property: background color;
```

Comment:
you can check on dev tools how work your styles
<img width="343" alt="Screenshot 2022-07-20 at 18 42 20" src="https://user-images.githubusercontent.com/47606379/180025241-3673d600-cb95-4cf6-9e8b-233c7322df1e.png">